reference-implementation/test/readable-stream-templated.js

```diff
@@ -54,13 +54,39 @@ templatedRSTwoChunksClosed('ReadableStream (two chunks enqueued, then closed)',
 );
 
 templatedRSTwoChunksClosed('ReadableStream (two chunks enqueued async, then closed)',
-  () => new ReadableStream({
-    start(c) {
-      setTimeout(() => c.enqueue(chunks[0]), 10);
-      setTimeout(() => c.enqueue(chunks[1]), 20);
-      setTimeout(() => c.close(), 30);
+  () => {
```

Comment:
Right. As this is expanded into various test cases, I chose to just make this stop enqueuing when cancel() is invoked.